Britannia/common/on_action/BTWK_game_start.txt

```diff
@@ -878,3 +880,17 @@ BTWK_de_jure_setup = {
 		}
 	}
 }
+
+#############################################################
+
+BTWK_story_trigger_on_game_start_479 = {
+	effect = {
+		set_global_variable = uther_is_dead
```

Comment:
Looks like this variable is actually used as a "uther_is_not_dead" variable - probably easier to just invert it wherever it's used (i.e. don't set it here, use NOT = { has_global_variable = uther_is_dead } to check that it's not there, set the variable instead of removing it when the character dies. etc)

Reply:
Quite a roundabout way of figuring out a more efficient solution, I can do it that way though sure. The initial naming & value convention of the variable worked for me (0 = off, not dead, 1 = on, dead), though it didn't matter so much given the prior changes you suggested. Probably should have commented why I had made the naming choice but it hadn't crossed my mind it would be an issue so, oh well.